github/code_scanning.go

```diff
@@ -391,7 +391,7 @@ func (s *CodeScanningService) UploadSarif(ctx context.Context, owner, repo strin
 		return nil, nil, err
 	}
 
-	// This will always return an error without unmarshalling the data
+	// This will always return an error without unmarshaling the data
```

Comment:
Please revert unrelated change. It's better to create a separate PR that fixes all `unmarshalling` words altogether.

Reply:
This change was explicitly requested by @gmlewis.

Reply:
And AFAIK it does fix all of the related typos.

Reply:
I created a PR to address this #3441
Would you mind a review?